modules/statics/src/networks.ts

```diff
@@ -535,7 +535,7 @@ class EosTestnet extends Testnet implements AccountNetwork {
 class Hedera extends Mainnet implements AccountNetwork {
   name = 'Hedera';
   family = CoinFamily.HBAR;
-  explorerUrl = 'https://explorer.kabuto.sh/mainnet/transaction/';
+  explorerUrl = 'https://app.dragonglass.me/hedera/search?q=';
```

Comment:
We can also use https://hashscan.io, just saying. But anything is fine. Please fix as per @mukeshsp 's comment.

Reply:
@ajays97  can you test for our testnet txn hashes, the hashscan says it is not found.